packages/gasket-plugin-morgan/lib/index.js

```diff
@@ -0,0 +1,24 @@
+const morgan = require('morgan');
```

Comment:
Could you add an `index.d.ts` as well so the new Gasket config properties are documented?

Reply:
Done, please take a look